### PR TITLE
Fix lttng-tools multilib issues

### PR DESCRIPTION
**Pull request by kergoth**
* Patch lttng-tools to obey its configure arguments for consumerd paths for multilibs
* Adjust lttng-tools packaging to avoid 32-bit binaries overwriting 64-bit ones

### CODE REVIEW

File: meta-sokol-flex-staging/recipes-kernel/lttng/lttng-tools_%.bbappend [hunk 1]
```diff
@@ -1,3 +1,11 @@
+# ---------------------------------------------------------------------------------------------------------------------
+# SPDX-License-Identifier: MIT
+# ---------------------------------------------------------------------------------------------------------------------
+
+FILESEXTRAPATHS:prepend := "${THISDIR}/lttng-tools:"
```
Comment (abelal) [inline meta-sokol-flex-staging/recipes-kernel/lttng/lttng-tools_%.bbappend:5]: should we be using the staging featureoverride here? and in the other places?

File: meta-sokol-flex-staging/recipes-kernel/lttng/lttng-tools_2.13.11.bb
Comment (abelal) [inline meta-sokol-flex-staging/recipes-kernel/lttng/lttng-tools_2.13.11.bb:1]: Is this an upgrade? The commit doesn't talk about this

File: meta-sokol-flex-staging/recipes-kernel/lttng/lttng-tools_%.bbappend (hunk 1 above)
Comment (kergoth) [reply meta-sokol-flex-staging/recipes-kernel/lttng/lttng-tools_%.bbappend:5]: Yes, thanks!

File: meta-sokol-flex-staging/recipes-kernel/lttng/lttng-tools_2.13.11.bb
Comment (kergoth) [reply meta-sokol-flex-staging/recipes-kernel/lttng/lttng-tools_2.13.11.bb:1]: Unintentional inclusion, thanks for spotting. I upgraded both components in testing, but weren't the critical elements. Will fix!

**Review by abelal — APPROVED** [review]

**Pull request merged**